pages/bookmarks.jsx

```diff
@@ -147,4 +165,15 @@ export default React.createClass({
       </div>
     );
   }
-});
+}
+
+Bookmarks.propTypes = {
```

Comment:
only `this.props.router.location` is used in this component so I think we can change the this part to just

```js
Bookmarks.propTypes = {
  router: PropTypes.shape({
    location: PropTypes.object
  })
};
```

Reply:
@mmmavis I've fixed the proptypes.